ansible/kubernetes.yml

```
@@ -10,7 +10,6 @@
   tasks:
     - name: Annotate nodes
       command: "kubectl annotate node --overwrite {{ inventory_hostname }} {{ item.key }}={{ item.value }}"
-      when: node_annotations is defined
-      with_dict: node_annotations
+      with_dict: node_annotations | default({})
```

```suggestion
      with_dict: "{{ node_annotations }}"
```

As in https://github.com/wireapp/wire-server-deploy/pull/363/commits/5326661ae72649e2d61bbf568375550b9e17a1a1

Which I forgot to cherry-pick to develop